# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../jahshaka/include.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- jahshaka/source/Jahshaka/JahLibraries/jahformatter/valueFormatters.cpp_orig	2006-05-17 14:52:37.368546250 +0200
+++ jahshaka/source/Jahshaka/JahLibraries/jahformatter/valueFormatters.cpp	2006-05-17 14:53:30.731881250 +0200
@@ -6,6 +6,7 @@
 
 // std
 #include <cmath>
+#include <cstdlib>
 
 TimecodeValueFormatter::TimecodeValueFormatter()
     : m_fps( 30 ),
--- jahshaka/source/OpenLibraries/openmedialib/mediaobject/mediaobject.cpp_orig	2006-05-07 22:39:17.000000000 +0200
+++ jahshaka/source/OpenLibraries/openmedialib/mediaobject/mediaobject.cpp	2006-05-17 13:52:13.174048250 +0200
@@ -8,6 +8,7 @@
 *******************************************************************************/
 
 #include "mediaobject.h"
+#include "mediaobject_.h"
 
 #ifdef JAHMPEGDECODER  
 #include "mediaobject_mpeg.h"
